diff --git a/src/hyperbus_delay.sv b/src/hyperbus_delay.sv
index ae1badc..5128cd2 100644
--- a/src/hyperbus_delay.sv
+++ b/src/hyperbus_delay.sv
@@ -15,8 +15,10 @@ module hyperbus_delay (
     generic_delay_D4_O1_3P750_CG0 i_delay (
         .clk_i      ( in_i      ),
         `ifndef TARGET_GF22
+        `ifndef TARGET_SYNTHESIS
         .enable_i   ( 1'b1      ),
         `endif
+        `endif
         .delay_i    ( delay_i   ),
         .clk_o      ( out_o     )
     );
